common/config.gni

```diff
@@ -16,6 +16,9 @@ declare_args() {
 
   # The runtime mode ("debug", "profile", "release", "dynamic_profile", or "dynamic_release")
   flutter_runtime_mode = "debug"
+
+  # Whether or not to use the GLFW desktop shell implementation.
+  use_glfw_desktop = is_linux || is_win
```

Comment:
This variable is not common to all targets. Please move this into a new config.gni file in `platform/foo` with its own declare_args block.